VisualPinball.TableScript/Program.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.IO;
+using VisualPinball.Engine.VPT.Table;
+
+namespace VisualPinball.TableScript
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			// validate inputs
+			try {
+				if (args.Length < 1) {
+					throw new ArgumentException("USAGE: VisualPinball.TableScript <input file or folder> [<output folder>]");
+				}
+
+				// input file(s)
+				if (!FileOrDirectoryExists(args[0])) {
+					throw new ArgumentException($"Non-existent input path: \"{args[0]}\".");
+				}
+				var inputAttr = File.GetAttributes(args[0]);
+				var inputFiles = (inputAttr & FileAttributes.Directory) == FileAttributes.Directory
+					? Directory.GetFiles(args[0], "*.vpx")
+					: new[] {args[0]};
+				if (inputFiles.Length == 0) {
+					throw new ArgumentException($"No .vpx files found at \"{args[0]}\"");
+				}
+
+				// output folder
+				string outputDir = null;
+
+				if (args.Length > 1) {
+					outputDir = args[1];
+					if (!Directory.Exists(outputDir)) {
+						throw new ArgumentException($"Invalid output folder \"{outputDir}\".");
+					}
+				}
+
+				foreach (var inputFile in inputFiles) {
+					Console.WriteLine($"Processing \"{inputFile}\"...");
+
+					var inputTable = TableLoader.Load(inputFile);
```

Comment:
You can add `false` here as second parameter so it doesn't load the game items, which should be a bit faster. You could even add another parameter to skip textures and sounds for extreme speed.

Reply:
awesome. Added `false`!